charts/multicluster-controlplane/templates/deployment.yaml

```diff
@@ -32,13 +32,18 @@ spec:
         {{- if eq .Values.enableSelfManagement true }}
         - "--self-management"
         {{- end }}
+        {{- if eq .Values.enableDelegatingAuthentication true }}
+        - "--delegating-authentication"
```

Comment:
Can you write a simple document how to use it in OpenShift cluster?

Reply:
yeah, I will open a new pr to add the document